Gopkg.lock

```diff
@@ -183,6 +207,12 @@
   revision = "2f5df55504ebc322e4d52d34df6a1f5b503bf26d"
 
 [[projects]]
+  branch = "master"
```

Comment:
Can we remove this from the PR? If it's necessary to update, please stacks the PRs.